components/brave_wallet_ui/page/screens/send/components/select-token-modal/select-token-modal.tsx

```diff
@@ -239,6 +263,11 @@ export const SelectTokenModal = React.forwardRef<HTMLDivElement, Props>(
                 onClick={() => onSelectSendAsset(token, account)}
                 key={`${token.contractAddress}-${token.chainId}-${token.tokenId}`}
                 balance={getBalance(account, token)}
+                spotPrice={
+                  spotPriceRegistry
+                    ? getTokenPriceAmountFromRegistry(spotPriceRegistry, token)
+                    : Amount.empty()
+                }
```

Comment:
pass in a string instead. since creating a new `Amount` instance will cause the component to always rerender

Reply:
fixed here: https://github.com/brave/brave-core/pull/18919/commits/47b342b19146b7be397f52fae71248fb56bf4430

Also checked that there are no other cases where we pass `Amount` object.